tools/src/main/scala/caliban/tools/compiletime/CompileTime.scala

```diff
@@ -0,0 +1,44 @@
+package caliban.tools.compiletime
+
+import caliban.GraphQL
+import caliban.tools.Codegen.GenType
+import caliban.tools.{ Codegen, Options, SchemaLoader }
+import zio.{ ExitCode, Task, URIO, ZEnv }
+
+object CompileTime {
+
+  def generateClient[R](args: List[String], api: GraphQL[R]): URIO[ZEnv, ExitCode] =
+    args match {
+      case toPath :: packageName :: clientName :: _ =>
+        (
+          for {
+            _      <- Task.unit
```

Comment:
You can simplify this slightly, removing `Task.unit` and apply `exitCode` to `generate`.

Reply:
Yup! Done. Thanks :)